### ISSUE TITLE
Locale selector:- Keyboard - Select region modal dialog

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#1769]
Issue :- Unable to close the dialog by pressing escape key.

Expected:- Apart from close button, user would also expect the dialog to be closed by pressing the "escape" key.

URL:- http://ibmdotcom-nextjs-test.mybluemix.net/learn.html

**Comment by jeffchew** [comment]
From the PR:

https://github.com/carbon-design-system/ibm-dotcom-library/pull/2046#issuecomment-613609879

```@chetankan @jeffchew @guilhermelMoraes Sorry I'm a little late to this but I wasn't aware of #1769. I'm not sure this is the correct interaction. This code is working in the LocaleModal story, but will fail in the Footer story since esc closes the entire modal, which is pretty much the standard behavior for all modals.

We should either trap the modal esc while filtering country/lang, or better yet, leave this as it is. This isn't an accessibility issue. What do you guys think?```

I'm inclined to leave this alone, thoughts?

cc: @ljcarot @kennylam @chetankan @Praveen-sr @guilhermelMoraes

**Comment by Praveen-sr** [comment]
@RobertaJHahn Ram re tested this on staging and Learn page, It is working fine. We are able to close the dialog by pressing escape key.

**Comment by jeffchew** [comment]
Per @Praveen-sr 's comment we are closing this issue.
